### ISSUE TITLE
delaying screenshot?

### ISSUE THREAD

**Issue by benhalverson** [devopsgroup-io/siteshooter#77]
@stevebritton 
Is it possible to delay the screenshot?
When I'm taking screenshots of overlays and modals it sometimes takes the screenshot before everything is displayed.
Here's an example
![screen shot 2017-09-28 at 11 14 54 am](https://user-images.githubusercontent.com/7907232/30983101-4c32e714-a43e-11e7-8093-45a9a10ed283.png)

**Comment by stevebritton** [comment]
Hey @benhalverson,

Have you tried working with the [delay setting in your siteshooter.yml file](https://github.com/devopsgroup-io/siteshooter/blob/master/siteshooter.yml#L15)? 

Also, if you're triggering overlays in an inject.js file, have you tried doing a setTimeOut?

**Comment by benhalverson** [comment]
@stevebritton yes I've tried both the yml option and setting a timeout in the inject.js file

**Comment by stevebritton** [comment]
@benhalverson,

I put together a quick example that will open a modal, start a timer within the model, and set the timer to the same value as the screen delay option and everything seems to be working as expected.

Example:
https://github.com/devopsgroup-io/siteshooter/tree/master/examples/screenshot-delay

I'm wondering if this is an async issue?
